frontend/src/components/Tree/Tree.jsx

```diff
@@ -100,7 +109,16 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick, onRightClick) {
       `
     )
     .attr("fill", (d) => (d.data.numChildren === 0 ? "#b30000" : "#555"))
-    .attr("r", 6);
+    .attr("r", 6)
+    .attr("opacity", (d) => {
+      if (hoveredNodeLink === "") {
+        return 1;
+      } else if (d.data.url === hoveredNodeLink) {
+        return 1;
+      } else {
```

Comment:
You can remove the else clause here and just return 0.25

Reply:
fixed